src/forgot-password/ForgotPasswordPage.jsx

```diff
@@ -25,22 +31,32 @@ import { INTERNAL_SERVER_ERROR } from '../login/data/constants';
 
 const ForgotPasswordPage = (props) => {
   const { intl, status } = props;
-  let invalidEmailMessage;
 
-  const validateEmail = (e, setFieldValue) => {
-    invalidEmailMessage = intl.formatMessage(messages['forgot.password.page.invalid.email.message']);
-    const regex = new RegExp(VALID_EMAIL_REGEX, 'i');
+  const platformName = getConfig().SITE_NAME;
+  const emailHelpMessage = (
+    <FormattedMessage
+      id="forgot.password.email.help.text"
+      defaultMessage="The email address you used to register with {platformName}."
```

Comment:
Move this into `messages.jsx` and then you can assign `helpText=intl.formatMessage(messages['forgot.password.email.help.text'])` and no need to create a variable.

Reply:
You can pass the values to `intl.formatMessage()` as well like we are doing here. https://github.com/edx/frontend-app-authn/blob/667389b1312375e3a08c546e8de138882b127342/src/login/LoginFailure.jsx#L43